.chloggen/fix_preserve-metric-names-request-split.yml

```diff
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
This file needs to have the extension `.yaml`, not `.yml`.

Reply:
@dmathieu 
Thanks for the review. I've updated the file extension to .yaml